docs/_docs/04_reference/cli-options.md

```diff
@@ -1082,14 +1084,23 @@ that Terragrunt invokes the module, so that you can debug issues with the terrag
 
 When passed it, sets logging level for terragrunt. All supported levels are:
 
-- `panic`
```

Comment:
We don't have logs with `panic` or `fatal` levels, so specifying any of these levels just disabled the logs at all. If people used this approach to disable logs, then yes, this is a breaking change and now they will return error.
```
--terragrunt-log-level, invalid level "fatal", supported levels: stderr, stdout, error, warn, info, debug, trace`
```

Reply:
Would it be possible to preserve the log levels just in-case, then? 

I think some users probably are taking that approach to avoiding logs. Even if someone currently sets `TERRAGRUNT_LOG_LEVEL=trace` because they don't like what we log, they should still see OpenTofu/Terraform output.

Reply:
> Even if someone currently sets TERRAGRUNT_LOG_LEVEL=trace

The `trace` level includes all log levels, so it is the most detailed log, and always has been.

> they should still see OpenTofu/Terraform output.

Specifying `stdout/stderr` has the same effect as `fatal/panic`, only now the level names make more sense.

All that can be done is not to output an error if the user specifies `fatal/panic` commands, i.e. automatically set the `stdout/stderr` level. Is that what you looking for?

Reply:
> Specifying stdout/stderr has the same effect as fatal/panic, only now the level names make more sense.

Let me clarify:
* Before new logs it disables TG logging at all, I don't mean forwarding TF output directly
* With new logs (main branch) it disables TG logging at all levels except `stdout`

Reply:
> I would suggest adding a new none level to non-ambiguously disable all logging, including stdout/stderr.

Or maybe a better solution would be the new `--terragrunt-log-disable` flag.

Reply:
Sorry, I meant `panic` earlier. I'm saying that right now, there is no `--terragrunt-log-disable` flag, so there's a chance that people out there have set the log level to `panic` or `fatal` to avoid logging anything from Terragrunt on purpose, but still want to see OpenTofu/Terraform stdout/stderr.

I don't want their workflows to break with this change, so I don't want to remove `panic` and `fatal`, as that doesn't really buy us anything, and (correct me if I'm wrong), I don't think it adds much burden to just leave it in.

The quickest way to address this recommendation would be to leave in `panic` and `fatal`, but to just have `stdout` and `stderr` be above them if that's feasible.

I'm fine with adding a log level of `none`, but I don't think we need to do that here. I think the average user would expect that Terragrunt still emits TF stdout/stderr if they set a flag like `--terragrunt-log-disable`. I don't think they really think of that as "logging" in the context of Terragrunt.

Reply:
If users use `panic/fatal`, it means that there is no functionality to disable logs and they found a workaround way, otherwise there is simply no point in using these levels.

I suggest to add a new `--terragrunt-log-disable` flag, that will disable _**TG**_ logs and enable `--terragrunt-forward-tf-stdout`, in this way users will be see only raw TF output without `time/level/prefix/msg`. Terraform also doesn't see stdout as logs, for this they have a separate env var TG_LOG=debug/info/warn/error.

If someone will still use `panic/fatal`, TG will not terminate with the error, instead print WARN message and enable `--terragrunt-log-disable`, `--terragrunt-forward-tf-stdout`. But `panic/fatal` will NOT be printed in help, as it now in the main branch.

```
Sets the logging level for Terragrunt. Supported levels: panic, fatal, error, warn, info, debug, trace. (default: info)
```

With this approach, we preserve the same behavior as was before [v0.67.0](https://github.com/gruntwork-io/terragrunt/releases/tag/v0.67.0) release.

What do you think? My goal is to suggest the best solution, the final word is yours. Make the final decision and I will implement it.

Reply:
> If someone will still use `panic/fatal`, TG will not terminate with the error, instead will ~~print WARN message and~~ enable `--terragrunt-log-disable`, `--terragrunt-forward-tf-stdout` and `panic/fatal` will NOT be printed in help, as it's now in the main branch.

Updated

Reply:
Ya, that works for me. As long as nobody gets an error for setting an old value, we're good.

Reply:
Done.